tapir/accounts/templates/registration/password_reset_confirm.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% load bootstrap4 %}
+{% load static %}
+
+{% block content %}
+  {% if validlink %}
+    <div class="card mt-4">
+      <h5 class="card-header">Set a new password</h5>
```

Comment:
Use {% trans %} tags here (and everywhere else)

Reply:
Thank you for pointing that out, the translation framework is now used in the whole password reset flow.